Mathlib/Topology/Algebra/Field.lean

```diff
@@ -91,6 +92,11 @@ def affineHomeomorph (a b : 𝕜) (h : a ≠ 0) : 𝕜 ≃ₜ 𝕜 where
     exact mul_div_cancel_left₀ x h
   right_inv y := by simp [mul_div_cancel₀ _ h]
 
+theorem affineHomeomorph_image_Icc {𝕜 : Type*} [LinearOrderedField 𝕜] [TopologicalSpace 𝕜]
+    [TopologicalRing 𝕜] (a b c d : 𝕜) (h : 0 < a) :
+    affineHomeomorph a b h.ne.symm '' Set.Icc c d = Set.Icc (a * c + b) (a * d + b) := by
+  simp [h]
+
```

Comment:
Could you please add the analogous lemmas for `Set.Ico`, `Set.Ioc`, and `Set.Ioo`?